packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -91,13 +93,15 @@
 			{param handleCloseMobileSearchClick_: $handleCloseMobileSearchClick_ /}
 			{param handleCreationButtonClicked_: $handleCreationButtonClicked_ /}
 			{param handleFilterDoneButtonClick_: $handleFilterDoneButtonClick_ /}
+			{param handleInfoButtonClicked_: $handleInfoButtonClicked_ /}
 			{param handleOpenMobileSearchClick_: $handleOpenMobileSearchClick_ /}
 			{param handleSearchSearchClick_: $handleSearchSearchClick_ /}
 			{param handleSelectPageCheckboxChanged_: $handleSelectPageCheckboxChanged_ /}
 			{param handleSortingButtonClicked_: $handleSortingButtonClicked_ /}
 			{param handleViewTypeClicked_: $handleViewTypeClicked_ /}
 			{param hideFiltersDoneButton: $hideFiltersDoneButton /}
 			{param id: $id /}
+			{param infoButton: $infoButton /}
```

Comment:
For consistency with the rest of the params I think this should be named `showInfoButton`